libs/application/templates/parental-leave/src/lib/answerValidationSections/utils.ts

```diff
@@ -271,3 +272,94 @@ export const validatePeriod = (
     }
   }
 }
+
+export const validatePeriodResidenceGrant = (
+  actualBirthDay: string,
+  expectedBirthDay: string,
+  multipleBirths: string,
+  dateFrom: string,
+  dateTo: string,
+) => {
+  const birthDayConverted = convertBirthDay(actualBirthDay)
+
+  const birthDate = new Date(
+    birthDayConverted.year,
+    birthDayConverted.month,
+    birthDayConverted.date,
+  )
+
+  const expBirthDate = expectedBirthDay.split('-').map((item) => Number(item))
+  const from = dateFrom.split('-').map((item) => Number(item))
+  const to = dateTo.split('-').map((item) => Number(item))
+
+  if (from.length !== 3 && to.length !== 3 && expBirthDate.length)
+    return {
+      field: 'dateFrom',
+      error:
+        parentalLeaveFormMessages.residenceGrantMessage
+          .residenceGrantGenericErrorMessage,
+    }
+  const expectedBirthDate = new Date(
+    expBirthDate[0],
+    expBirthDate[1] - 1,
+    expBirthDate[2],
+  )
+  const toDate = new Date(to[0], to[1] - 1, to[2])
+  const fromDate = new Date(from[0], from[1] - 1, from[2])
+
+  const expBirthDateMinus =
+    multipleBirths === 'yes'
+      ? subDays(expectedBirthDate, 28)
+      : subDays(expectedBirthDate, 14)
```

Comment:
Should use constant/type YES insteads of multipleBirths === 'yes'

Reply:
I am taking out this validation.